app/exec/extension/install.ts

```diff
@@ -65,15 +67,15 @@ export class ExtensionInstallResult {
 }
 
 export class ExtensionInstall extends extBase.ExtensionBase<ExtensionInstallResult> {
-	protected description = "Install a Visual Studio Services Extension to a list of VSTS Accounts.";
```

Comment:
Should be "Install an extension to one or more Team Services accounts or TFS team project collections."